editor/editor_toaster.h

```diff
@@ -33,6 +33,7 @@
 
 #include "scene/gui/box_container.h"
 #include "scene/gui/button.h"
+#include "scene/gui/panel_container.h"
```

Comment:
It's also in the .cpp, so the .cpp one can be removed (or the header one can be forward declared).